.github/workflows/ci.yml

```diff
@@ -175,7 +175,8 @@ jobs:
   verify-projects:
     name: Verify project
     needs: [ package, unit-tests ]
-    uses: beeware/.github/.github/workflows/app-create-verify.yml@main
+    # uses: beeware/.github/.github/workflows/app-create-verify.yml@main
+    uses: beeware/.github/.github/workflows/app-create-verify.yml@qt-requires
```

Comment:
Just leaving this here as a reminder to revert this change after the .github PR is merged.